big_tests/tests/gdpr_SUITE.erl

```diff
@@ -290,11 +307,11 @@ retrieve_and_validate_personal_data(Alice, Config, FilePrefix, ExpectedHeader, E
              })
     end.
 
-csv_to_maps(ExpectedHeader, [HeaderRow | [Rows]]) ->
+csv_to_maps(ExpectedHeader, [ExpectedHeader | Rows]) ->
```

Comment:
Such fixes should be handled separately, or even directly in base branch, as it affects the logic of all other branches.